### DISCUSSION TITLE
en/lc/1609/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#2392] [Announcements]
# en/lc/1609/

LeetCode solutions in any programming language

https://leetcode.doocs.org/en/lc/1609/

**Comment by nghgkim** [comment]
Java's Solution 1, if prev = even? 0 : 1000000, 1 testcase will be wrong, you have to change 1000000 to 1000001 to pass all testcases

**Reply by yanglbme** [reply]
Thank you for your feedback. I have already fixed this issue.
